common/lib/xmodule/xmodule/assets/vertical/public/js/vertical_student_view.js

```diff
@@ -28,7 +28,7 @@ window.VerticalStudentView = function(runtime, element) {
         function(idx, block) {
             var blockKey = block.dataset.id;
 
-            if (block.dataset.completableByViewing === undefined) {
+            if (block.dataset.completableByViewing === undefined || block.dataset.completableByViewing === '') {
```

Comment:
According to https://www.sitepoint.com/javascript-truthy-falsy/, javascript treat both `undefined` and `''` falsy.
Therefore, what is the reason not to use `if (!block.dataset.completableByViewing)`?

Reply:
Good point.